src/components/common/Item.js

```diff
@@ -205,6 +205,13 @@ const Item = ({ id, isChildren, showPinnedOnly }) => {
       );
     }
 
+    case ITEM_TYPES.SHORTCUT: {
+      if (item.extra?.shortcut?.target) {
+        return <Item isChildren id={item.extra?.shortcut?.target} />;
+      }
+      return null;
```

Comment:
If the following makes sense:
```suggestion
      return (
          <Alert severity="error">{t('An unexpected error occured.')}</Alert>
        );
```